src/openms/include/OpenMS/COMPARISON/CLUSTERING/GridBasedClustering.h

```diff
@@ -539,17 +539,23 @@ namespace OpenMS
     {
       int A1 = c1.getPropertyA();
       int A2 = c2.getPropertyA();
-      std::vector<int> B1 = c1.getPropertiesB();
-      std::vector<int> B2 = c2.getPropertiesB();
 
       // check if any of the properties A and B is not set i.e. =-1
```

Comment:
Please update comment.

// Check if the properties A of both clusters have been set or not. (i.e. =-1 if not set)